doc/e2ee.md

```diff
@@ -54,10 +66,8 @@ At a high level the encrypted frame format looks like this:
    | |                                                       | |
    | |                                                       | |
    | |                                                       | |
-   +^+-------------------------------------------------------+ +
-   | |                 Authentication Tag                    | |
-   | +---------------------------------------+-+-+-+-+-+-+-+-+ |
-   | |    CTR... (length=LEN + 1)            |R|LEN  |KID    | |
+   | +------------------------+-------------------------+----- |
+   | | GCM... (length=LEN)    | IV...(length=IV_LENGTH) |KID | |
```

Comment:
There is no such thing as a GCM field.

Reply:
Do we no longer have the LEN? Also, let's keep the reserved field, while we're at it.